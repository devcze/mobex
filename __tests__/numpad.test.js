/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Numpad from '../components/numpad'

describe('Numpad', () => {
  it('Numpad render', () => {
    const component = renderer.create(<Numpad/>);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
