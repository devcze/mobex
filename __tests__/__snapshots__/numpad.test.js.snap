// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Numpad Numpad render 1`] = `
<div
  className="numpad__Numpad-i65ovq-0 dWifEs"
>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    1
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    2
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    3
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    4
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    5
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    6
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    7
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    8
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    9
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    .
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    0
  </div>
  <div
    className="numpad-button__NumpadButton-sc-1n447cv-0 fATyam"
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    style={
      Object {
        "backgroundColor": "white",
        "color": "#444444",
      }
    }
  >
    â†¤
  </div>
</div>
`;
